DependencyInjection/Configuration.php

```diff
@@ -26,7 +26,11 @@ public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder();
         $treeBuilder->root('cmf_routing_auto')
+            ->addDefaultsIfNotSet()
             ->children()
+                ->scalarNode('adapter')
+                    ->defaultValue('doctrine_phpcr_odm')
+                ->end()
```

Comment:
I think most of the time, when a node has only one method call (in this case `defaultValue()`), everything is put on one line. As soon as you have multiple method calls, it's indented like this.

At the moment, there is no `info()`, but if you add it, putting them on multiple lines would be better.